Makefile

```diff
@@ -5,7 +5,7 @@ EXTRACTED_RULE_FILE = test/promql/extracted-rules.yaml
 RULE_FILE = prometheus/base/prometheus.rules.yaml
 
 .PHONY: all
-all: prepare lint test_rules pint_lint
+all: prepare install_pipenv sync_pipenv lint test_rules pint_lint lint_yamls
```

Comment:
Installing pipenv is a prerequisite you put on the contributing guide so you don't need to include it in make all. It will just run in ci.

Reply:
I think it's best if I remove it from the contributing guide and not modify the makefile, because it will helpful for the developers to just run one command ‘make all’. In that way, it will be one less step for the developers. Whats your suggestion?

Reply:
I'd prefer the makefile not to hide installing of packages that has effect on the user's machine outside of the local repo and virtual env. 

@eisraeli @amisstea @avi-biton @hmariset any thoughts?

Reply:
@Omeramsc

Reply:
I agree with @yftacherzog 
Please don't change anything on the user's machine outside of the  local repo and virtualenv.  It's too intrusive.

Reply:
I understand, I have removed the `installing pipenv` from the Makefile.

> I agree with @yftacherzog Please don't change anything on the user's machine outside of the local repo and virtualenv. It's too intrusive.

Got it, I have removed the `installing pipenv` from the Makefile.